src/main/java/com/db/transform/service/TransformService.java

```diff
@@ -0,0 +1,63 @@
+package com.db.transform.service;
+
+import com.db.transform.entity.Trade;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class TransformService {
+
+    WebClient webClient;
```

Comment:
This code will need to be refactored later to separate webClient to a repository and replace it with the correct things, so i think we can exclude this from this pr for now.